packages/storybook/src/stories/ino-nav-menu/ino-nav-menu.stories.ts

```diff
@@ -0,0 +1,77 @@
+import { Components } from '@inovex.de/elements';
+import { Meta } from '@storybook/web-components';
+import { html } from 'lit-html';
+import { TemplateGenerator } from '../template-generator';
+import { decorateStoryWithClass } from '../utils';
+import './ino-nav-menu.scss';
+
+export default {
+  title: `Structure/ino-nav-menu`,
+  component: 'ino-nav-menu',
+  decorators: [(story) => decorateStoryWithClass(story, 'story-ino-nav-menu')],
+  args: {
+    sections: ['Section 1', 'Section 2', 'Section 3'],
+    activeSection: 'section-1',
+    menuTitle: 'Sections',
```

Comment:
`menuTitle`, `intersectionObserverConfig` and `scroll-offset` could be showcased separately in stories. The same applies for `intersectionObserverConfig` and `scroll-offset`. We try to showcase every prop within their own story.
The same goes for the `ino-nav-menu-section` prop `showTitle`